ctrlz/topics/assets/assets.go

```diff
@@ -12,6 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:generate $GOPATH/src/istio.io/istio/bin/go-bindata.sh --nocompress --nometadata --pkg assets -o assets.gen.go ./templates/...
+//go:generate ../../../scripts/go-bindata.sh --nocompress --nometadata --pkg assets -o assets.gen.go ./templates/...
```

Comment:
What is the root of this? Is it the invocation current-working directory, or is it he location of assets.go? (It looks like the latter).

Reply:
The current directory when this thing runs is set to the location of the .go file. So I can do "go generate ./..." from the root of the repo, or I can do "go generate" from the specific subdirectory and it works.